website/src/pages/features/index.tsx

```diff
@@ -48,35 +67,59 @@ function FeatureManageImages() {
 <div className="lg:flex-grow md:w-1/2 lg:pl-24 md:pl-16 flex flex-col md:items-start md:text-left items-center text-center">
   <h1 className="title-font sm:text-4xl text-3xl mb-4 font-medium text-gray-900 dark:text-white">Build, Pull and Push images</h1>
 
-<p className="leading-relaxed list-item">Build images from the tool.</p>
-<p className="leading-relaxed list-item">Pull and push images by managing registries.</p>
-<p className="leading-relaxed list-item">Run containers from these images</p></div>
+<p className="leading-relaxed list-item dark:text-gray-200 text-gray-800">Build images from the tool.</p>
+<p className="leading-relaxed list-item dark:text-gray-200 text-gray-800">Pull and push images by managing registries.</p>
+<p className="leading-relaxed list-item dark:text-gray-200 text-gray-800">Run containers from these images</p></div>
 
 </div>
 </section>);
 
 }
 
-
-
-function FeatureManageResources() {
+function FeatureManagementFromTrayIcon() {
   return (
     <div>
-      <section className="text-gray-600 dark:bg-zinc-800 dark:text-gray-40 body-font">
+      <section className="text-gray-600 bg-zinc-200 dark:bg-zinc-800 dark:text-gray-400 body-font">
         <div className="container mx-auto flex px-5 py-24 md:flex-row flex-col items-center">
           <div className="lg:flex-grow md:w-1/2 lg:pr-24 md:pr-16 flex flex-col md:items-start md:text-left mb-16 md:mb-0 items-center text-center">
             <h1 className="title-font sm:text-4xl text-3xl mb-4 font-medium text-gray-900 dark:text-white">
-              Manage Podman resources
+              Management from the tray icon
               <br className="hidden lg:inline-block" />
             </h1>
             <p className="leading-relaxed list-item dark:text-gray-200 text-gray-800">
-              View allocated memory, CPU and storage.
+              Check status and start/stop container engines.
             </p>
             <p className="leading-relaxed list-item dark:text-gray-200 text-gray-800">
-              Create new machine if needed
+              Create new machine if needed.
+            </p>
+            <p className="leading-relaxed list-item dark:text-gray-200 text-gray-800">
+              Start or stop Podman machines directly from the tray icon.
             </p>
           </div>
           <div className="lg:w-5/6 md:w-4/5 w-5/6 flex flex:col gap-10">
+            <ThemedImage
```

Comment:
I think the image does not look exactly to fit with the other ones in the page:
- it looks scaled up in resolution: maybe we can resize it?
<img width="1580" alt="Capture d’écran 2022-06-21 à 22 43 22" src="https://user-images.githubusercontent.com/1636769/174893922-6769bd28-e285-458d-8671-b30853bf87fd.png">
- I think it's missing ronded corners and shadows to be consistent. You can check on tailwind how to do add property to the class: https://tailwindcss.com/docs/border-radius and https://tailwindcss.com/docs/box-shadow

Reply:
I'm not sure about resizing as it will add more noise to the image. By noise I mean other information that is irrelevant since its a small section on the tray. I will still send alternative screenshots. I will work on the consistency of the image too.